examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -96,21 +96,22 @@ void OTAProviderExample::SetOTAFilePath(const char * path)
 }
 
 EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * commandObj, uint16_t vendorId, uint16_t productId,
-                                                   uint16_t imageType, uint16_t hardwareVersion, uint32_t currentVersion,
-                                                   uint8_t protocolsSupported, const chip::Span<const char> & location,
-                                                   bool clientCanConsent, const chip::ByteSpan & metadataForServer)
+                                                   uint16_t hardwareVersion, uint32_t softwareVersion, uint8_t protocolsSupported,
```

Comment:
Is there an issue filed to track updating the type of `protocolsSupported`?

Reply:
Yes, https://github.com/project-chip/connectedhomeip/issues/8605